src/internal/pctx/context.go

```diff
@@ -21,6 +23,11 @@ func Background(process string) context.Context {
 	return Child(ctx, process)
 }
 
+// Interactive returns a context for use in command-line apps.
+func Interactive() (context.Context, context.CancelFunc) {
+	return signal.NotifyContext(log.AddLogger(context.Background()), os.Interrupt)
```

Comment:
I was tired of copy-pasting this.  Also terminationSignals is very annoying to use, because it ignores C-\, so there is no way to shut down a broken program.  For that reason, we only grab interrupt here.

Reply:
It’s great to fix this instead of copy-pasting.

I guess it’s _okay_ not to take a context as an argument, since signals are process-global, and the user can always augment the context later — unless using another library which thinks its context should be global for another reason. Might still be better to take an argument.

What signal does `C-,` send in your terminal? `signals.TerminationSignals` is (on Unix) `SIGINT`, `SIGTERM`, `SIGQUIT` and `SIGHUP`. I guess your terminal probably sends `SIGQUIT`, which is normally bound to `C-\`? One can always shut down a broken program with `kill -9`, of course, but I see your point.

Maybe this should be smarter still:

- `SIGINT`/`SIGHUP` (normally on `C-c`): cancel the context
- `SIGQUIT`: `os.Exit(255)`

Reply:
I don't like messing with SIGQUIT.  SIGQUIT is how you kill stuff with your keyboard when all else fails.